packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -43,12 +45,41 @@ where
     T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone,
 {
     pub db: DB<T>,
+    cache_unrealized_balances: HashMap<Hash, Balance>,
     pub me: Address,
 }
 
 impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone> CoreEthereumDb<T> {
     pub fn new(db: DB<T>, me: Address) -> Self {
-        Self { db, me }
+        Self {
+            db,
+            cache_unrealized_balances: HashMap::new(),
```

Comment:
The naming sounds misleading now, as we are speaking about unrealized ticket values in this context.